src/components/MergeInfographic/Background.tsx

```diff
@@ -0,0 +1,91 @@
+import { createIcon } from "@chakra-ui/react"
+
+const [w, h] = [250, 110]
+export const Background = createIcon({
+  displayName: "MergeInfographicBackground",
+  viewBox: `0 0 ${w} ${h}`,
+  defaultProps: {
+    height: `${h}px`,
+    width: `${w}px`,
+    fill: "currentColor",
+  },
+  path: [
+    <g clip-path="url(#clip0_10_161)">
```

Comment:
All these attrs should be camelCased
```suggestion
    <g clipPath="url(#clip0_10_161)">
```

Reply:
Hah, that last step I sometimes forget when converting .svg's to React =)... patched.